<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1190</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Loan Search</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: #fbfbfb;
}

QDateEdit {
	border: none;
    border-bottom: 1px solid black;
    background-color: transparent;
    color: black;
}

QDateEdit:disabled {
	border: none;
    border-bottom: 1px solid lightgray;
    background-color: lightgray;
    color: lightgray;
}

QPushButton {
    background-color: #0077c2;
    color: white;
    border: 1px solid transparent;
    border-radius: 10px;
    padding: 5px 10px;
}

QPushButton:hover {
    border: 1px solid white;
}

QPushButton:pressed {
    background-color: #005f9e;
    padding-top: 6px;
    padding-bottom: 4px;
}

QPushButton:disabled {
    background-color: lightgray;
    color: gray;
    border: 1px solid gray;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="cancelPaymentButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cancel Payment</string>
    </property>
   </widget>
   <widget class="QTableView" name="repaymentScheduleTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>1171</width>
      <height>501</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #f1f1f1;
border: 1px solid transparent;
border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #2e9730; color: white;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Repayment Schedule</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QDateEdit" name="startDate">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
        QCalendarWidget QToolButton {
            background-color: blue; /* 연도와 월 버튼 배경을 파란색으로 설정 */
            color: white; /* 연도와 월 버튼 글자 색을 흰색으로 설정 */
            border: none; /* 테두리를 없앰 */
        }
        QCalendarWidget QToolButton::hover {
            background-color: darkblue; /* 커서를 올렸을 때 배경을 진한 파란색으로 설정 */
        }
     </string>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::UpDownArrows</enum>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>16</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
     </font>
    </property>
    <property name="text">
     <string>~</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="endDate">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
        QCalendarWidget QToolButton {
            background-color: blue; /* 연도와 월 버튼 배경을 파란색으로 설정 */
            color: white; /* 연도와 월 버튼 글자 색을 흰색으로 설정 */
            border: none; /* 테두리를 없앰 */
        }
        QCalendarWidget QToolButton:hover {
            background-color: darkblue; /* 커서를 올렸을 때 배경을 진한 파란색으로 설정 */
        }
     </string>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::UpDownArrows</enum>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="detailsButton">
    <property name="geometry">
     <rect>
      <x>1072</x>
      <y>60</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Details</string>
    </property>
   </widget>
   <widget class="QPushButton" name="paidButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Paid</string>
    </property>
   </widget>
   <widget class="QPushButton" name="overdueButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Overdue</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>startDate</tabstop>
  <tabstop>endDate</tabstop>
  <tabstop>cancelPaymentButton</tabstop>
  <tabstop>repaymentScheduleTable</tabstop>
  <tabstop>searchButton</tabstop>
  <tabstop>detailsButton</tabstop>
  <tabstop>paidButton</tabstop>
  <tabstop>overdueButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
